fs/expose/dokan/__init__.py

```diff
@@ -1069,11 +1069,11 @@ def __init__(self, fs, path, dokan_opts={}, nowait=False, **kwds):
             raise OSError("the dokan library is not available")
         _check_path_string(path)
         self.path = path
-        cmd = "try: import cPickle;\nexcept ImportError: import pickle as cPickle;\n"
-        cmd = cmd + "data = cPickle.loads(%s); "
+        cmd = "try: import pickle;\nexcept ImportError: import pickle as pickle;\n"
```

Comment:
Shouldn't this be `try: import cPickle as pickle; except ImportError: import pickle`